grails-forge-core/src/main/java/org/grails/forge/feature/database/H2.java

```diff
@@ -40,10 +40,20 @@ public String getDescription() {
     }
 
     @Override
-    public String getJdbcUrl() {
+    public String getJdbcDevUrl() {
         return "jdbc:h2:mem:devDb;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE";
     }
 
+    @Override
+    public String getJdbcTestUrl() {
+        return "jdbc:h2:mem:testDb;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE";
+    }
+
+    @Override
+    public String getJdbcProdUrl() {
+        return "jdbc:h2:./prodDb;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE";
```

Comment:
should these database files be under the build directory instead of in the project root?

Reply:
Ignore this, I see the prod one isn't drop/recreated.  so it's probably best at the project root.